src/index.ts

```diff
@@ -127,7 +180,8 @@ function getEmit(
     // Emit Javascript
     const output = instance.languageService.getEmitOutput(filePath);
 
-    loader.clearDependencies();
+    // Remove original dependency on resourcePath and replace with rawFilePath
```

Comment:
Of course, I should check if `rawFilePath === this.resourcePath`, and if so skip this step entirely.